src/app/Fake.Azure.CloudServices/CloudServices.fs

```diff
@@ -0,0 +1,97 @@
+﻿/// Contains tasks to package Azure Cloud Services.
+module Fake.Azure.CloudServices
+
+open System.IO
+open Fake.Core
+open Fake.IO
+open Fake.IO.Globbing.Operators
+
+/// Configuration details for packaging cloud services.
+[<CLIMutable>]
```

Comment:
Can we remove `CLIMutable` here?

Reply:
I copied all this from the `FakeLib` project.

Reply:
Yes I know and this helps a lot! I just like APIs to be more consistent in fake5 so we can improve the code while porting - this moves the project forward. I see that people want to use stuff on fake5. That’s why I said the items are suggestions just in case you agree with them and have a minute ;)